app/src/main/java/com/twilio/video/app/data/api/VideoAppServiceDelegate.kt

```diff
@@ -32,7 +32,7 @@ class VideoAppServiceDelegate(
     private val videoAppServiceProd: VideoAppService
 ) : TokenService {
 
-    override suspend fun getToken(identity: String?, roomName: String?): String {
+    override suspend fun getToken(identity: String?, roomName: String?, passcode: String?): String {
```

Comment:
See my comment above, but I think we should update the interface definition so the passcode is not referenced outside the community version